packages/ui/node/reporter.ts

```diff
@@ -59,6 +60,10 @@ export default class HTMLReporter implements Reporter {
     this.start = Date.now()
   }
 
+  async onFinishedReportCoverage() {
+    await prepareReportCoverageFolder(this.ctx)
+  }
+
   async onFinished() {
     const result: HTMLReportData = {
```

Comment:
This logic should be moved to `onFinishedReportCoverage` since the coverage still not generated:
- running the standalone html reporter script will include ui client in the coverage report
- we should remove the html folder on every run using standalone html reporter, otherwise the ui client from previous run will be included in the coverage report

Running this script from `examples/basic` folder:
`pnpm vitest --ui --open --coverage.enabled=true --coverage.reporter=html --reporter=html --coverage.reportsDirectory=./html/coverage/`

Running first time standalone html reporter (all good):
![imagen](https://github.com/user-attachments/assets/57376986-2fff-46bf-a126-ebcd54203cd0)

Running second time without cleanup html folder:
![imagen](https://github.com/user-attachments/assets/48278e73-fd76-46a1-afb0-bc697a0f683b)